src/components/_product/Description/Description.tsx

```diff
@@ -0,0 +1,27 @@
+'use client';
+import { useState } from 'react';
+import styles from './Description.module.scss';
+import type { Props } from './Description.types';
+
+const Description = ({ children }: Props) => {
+  const [selectedTab, setSelectedTab] = useState(0);
+
+  return (
+    <section className={styles['Description']}>
+      <div className={styles['tabs']}>
+        {children.map((_, i) => (
+          <button
+            key={i}
+            onClick={() => setSelectedTab(i)}
+            className={i === selectedTab ? styles['active'] : ''}
```

Comment:
I prefer working with data-* attributes for such an things. In my opinion, it look much clearner.

https://github.com/KryptonumDev/kierunek-dzierganie/blob/7664bb65bdc1467e75709c9736ae29f2231bc929/src/components/_global/TabSection/_Tabs.tsx#L24